VideoPlayer/VideoPlayer.js

```diff
@@ -2031,6 +2044,18 @@ const VideoPlayerBase = class extends Component {
 						>
 							{secondsToTime(this.state.sliderTooltipTime, durFmt)}
 						</FeedbackContent>
+						{
+							this.state.mediaControlsVisible ?
+								<Button
+									aria-label={backButtonAriaLabel == null ? $L('go to previous') : backButtonAriaLabel}
+									className={css.back}
+									icon="arrowhookleft"
+									iconFlip="auto"
+									onClick={onBack}
```

Comment:
This lets the event type not be `onBack`.
Let's use `this.handleBack` which look like
```
handleBack = this.handle(forwardCustom('onBack'));
```